corehq/apps/reports_core/filters.py

```diff
@@ -184,6 +181,7 @@ def year_param_name(self):
 
     @property
     def years(self):
+        from corehq.apps.userreports.reports.filters.values import SHOW_ALL_CHOICE
```

Comment:
These are bunch of circular import issues, as `userreports` module uses functions defined in `reports_core` module. This is the only place where `reports_core` uses functions defined in `userreports`